tools/ClassGen/NodeBuilder.cpp

```diff
@@ -352,23 +405,53 @@ void NodeBuilder::emitPrettyPrinter(std::ostream &os) const {
   }
 
   // Generate description for outputs.
+  // --> Fix outputs.
   for (const auto &op : nodeOutputs_) {
     os << "  db.addParam(\"" << op.second << "\", *(get" << op.second
        << "().getType()));\n";
   }
+  // --> variable outputs.
+  if (variableOutputMembers_.size()) {
+    os << "  unsigned_t variableOutIdx = " << nodeOutputs_.size() << ";\n";
+    for (const auto &op : variableOutputMembers_) {
+      os << "  for (unsigned i = 0; i < " << op.second << "_size_; i++) {\n"
+         << "   db.addParam(\"" << op.second << "\" + std::to_string(i),"
+         << " *(getNthResult(variableOutIdx + i).getType()));\n"
```

Comment:
Could you add comments? Are you trying to get Type of the i-th element of op.second variable size output? If so, maybe it is safer to call "get + op.second + NthResult(i)" instead of hard-coding the current mapping via a general getNthResult?

Reply:
yes, the type of i-th element of op.second variable size output.
On don't think it's sager since `getNthResult` will also manage the variability properly, but it can be more consistent with the rest of the code (that gets the size with the output name).
It would be : `"get" << op.second << "Nth(i)"`

Reply:
Comment added and use the specific value getter rather than the generic one.